src/modules/wallet/components/AllTokens/AllTokens.js

```diff
@@ -29,8 +28,13 @@ const AllTokens = ({ history }) => {
   const { t } = useTranslation();
   const timeout = useRef();
 
-  const [search, setSearch] = useState('');
+  const [searchToken, setSearch] = useState('');
   const { filters, setFilter } = useFilter({});
+  const { search } = useLocation();
+  const queryParams = new URLSearchParams(search);
+  const disableSend = Boolean(queryParams.get('disableSend'));
```

Comment:
Doesnt this query param contain a string representation of 'true' or 'false'? 
In that case we cannot rely on `Boolean(` since 
<img width="146" alt="image" src="https://github.com/LiskHQ/lisk-desktop/assets/8784876/99050c91-66bb-4309-9ce7-8d13eef7deed">

Better to do:
`const isSendDisabled = queryParams.get('disableSend') === 'true';`
🤔

Reply:
As you can see here: 
<img width="1713" alt="image" src="https://github.com/LiskHQ/lisk-desktop/assets/8784876/2ab98433-4209-4eca-a787-822cb0ae7865">